### DISCUSSION TITLE
Should repeated notes get tab added?

### DISCUSSION THREAD

**Discussion by jgadsden** [jgadsden/tin-whistle-tablature#43] [Ideas]
@Merrit  asked in issue #42 'Notes sometimes don't get tab added' about the absence of tablature for repeated notes

As far as I know it is convention not to repeat the tablature for a repeated note, so tablature only when the fingering changes, but is this correct?

**Comment by Merrit** [comment]
I am very new to both music and tin whistle, so that might very well be the norm.

I've been primarily going off the tabs from the youtuber CutiePie; in this instance specifically trying to make a better copy of one of her tabs:

![image](https://github.com/jgadsden/tin-whistle-tablature/assets/9575627/d514e9a0-0cc5-45c1-a663-458f2882b57f)

All of hers have had tabs even for repeated notes. So I just presumed that was the way it should be! :) Perhaps because they are aimed at absolute beginners.

Definitely something I can get used to if that is the expected way to do them, I was just confused and thought I'd found a bug.

**Comment by jgadsden** [comment]
Hello @Merrit 
Not at all, I think you may have spotted something missing from the tin whistle tabs
It could be that if the repeated note is the first in the bar, then it should be explicitly shown and not implied from the previous bar
I honestly do not know what it should be, but my guess is that CutiePie has it right

**Comment by Merrit** [comment]
Yeah, I can see that. Probably should especially be shown if it is on a new line, where it is less obvious:

![image](https://github.com/jgadsden/tin-whistle-tablature/assets/9575627/8dcdd7a6-e2a8-4009-8230-d452b42d69b2)

**Comment by jgadsden** [comment]
You make a very good point @Merrit , and I will see if I can fix this and release a new version of tinw whistle tabs using issue #42

**Comment by jgadsden** [comment]
Hello @Merrit  , looking at it this month - possibly with a fix too :)
